calliope/test/common/test_model/storage_capacity_ratio.yaml

```diff
@@ -0,0 +1,41 @@
+model:
+    name: Test model for storage capacity ratio
+    timeseries_data_path: 'timeseries_data'
+    subset_time: '2005-01-01'
+
+run:
+    mode: plan
+    solver: glpk
+    cyclic_storage: False
+
+techs:
```

Comment:
hmm, how does a model with only storage and demand work? There's no way for the system to get energy to begin with. Maybe add a supply tech?

Reply:
Mhm the whole point of a minimal example is to be minimal. The demand can be fed from a storage, as long as `cyclic_storage` is turned off, which it is. The tests prove it works.

Reply:
Quite. Although if we ever want these to become part of an examples library online, it does seem a little odd. I'm happy to have it remain as is, but maybe with a comment in the YAML next to the cyclic storage that turning it off is what makes the model feasible.